app/stories/components/InputChoiceGroup.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+
+import InputChoiceGroupComponent from '@components/InputChoiceGroup'
+
+export default {
+  title: 'Components',
+  component: InputChoiceGroupComponent,
+} as ComponentMeta<typeof InputChoiceGroupComponent>
+
+const Template: ComponentStory<typeof InputChoiceGroupComponent> = (args) => (
+  <InputChoiceGroupComponent {...args} />
+)
+
+export const InputChoiceGroup = Template.bind({})
+
+InputChoiceGroup.args = {
+  choices: [
+    {
+      checked: true,
+      handleChange: () => {},
```

Comment:
The radio buttons/checkboxes aren't selectable in the story. That seems wrong to me. 🤔